code/modules/hydroponics/seed_datums.dm

```diff
@@ -163,7 +163,7 @@
 	name = "tomato"
 	seed_name = "tomato"
 	display_name = "tomato plant"
-	mutants = list("bluetomato","bloodtomato")
+	mutants = list("bluetomato","bloodtomato","killertomato")
```

Comment:
The PR title is misleading with this, is this a autolathe update or xenobotany feature?

Reply:
This was actually an accident, I was sure it was reverted D: